lib/config-migrator.js

```diff
@@ -0,0 +1,59 @@
+/** @babel */
+
+import _ from 'lodash'
+import { heredoc } from './werkzeug'
+
+export default async function checkConfigAndMigrate () {
```

Comment:
Async in JS is implemented using Promises. You can `.then` from the sync side instead.